docker-compose-fullstack.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
```

Comment:
We should also add the celery worker/scheduler. It should be as easy as duping the updated `website` service and overriding the `command` in here.